src/plugins/console/public/application/containers/editor/legacy/console_editor/editor.tsx

```diff
@@ -68,7 +68,7 @@ const abs: CSSProperties = {
   right: '0',
 };
 
-const DEFAULT_INPUT_VALUE = `GET _search
+export const DEFAULT_INPUT_VALUE = `GET _search
```

Comment:
let's try not to change the access level of any code in "legacy", which will deprecate in the future. If you really need it, consider moving the const to non-legacy path.